src/components/IstioWizards/WizardActions.ts

```diff
@@ -1137,6 +1137,87 @@ export const buildAuthorizationPolicy = (
   return ap;
 };
 
+export const buildGraphSidecars = (namespace: string, graph: GraphDefinition): Sidecar[] => {
+  const denyAll: Sidecar = {
```

Comment:
I don't think the "denyAll" applies to the sidecars, as it's not like the AuthorizationPolicy, so I guess for Sidecars it can rely on the workloads only.

Reply:
I guess the name is wrong, that is true, but what I wanted to do is to create a general rule that will applies for future workloads, to restrict the egress traffic just to the own namespace.

But, thinking a little bit more, maybe it's not correct to take this assumption, when new workloads are created, user should update the traffic policies.

I'm gonna remove it.

Reply:
> I'm gonna remove it.

+1, we should not make that assumption but rather only reflect the state at the time this action is performed.